documentation/topics/migrations_and_tasks.md

```diff
@@ -74,9 +74,9 @@ Define a module similar to the following:
 ```elixir
 defmodule MyApp.Release do
   @moduledoc """
-  Houses tasks that need to be executed in the released application (because mix is not present in releases).
+  Housekeeping tasks that need to be executed in the released application (because mix is not present in releases).
```

Comment:
Tasks that need to be executed in the released application (because mix is not present in releases).